src/components/modals/TranscriptRatings.tsx

```diff
@@ -16,8 +16,8 @@ import './TranscriptRatings.css'
 interface ComponentState {
     ratePivot: string | undefined
     sourceRankMap: Map<string, RankCount[]>
-    unRatableMap: Map<string, number>    // Number conversations missing transcript for comparison
-    notRatedMap: Map<string, number>    // Number conversations that haven't been rated
+    unRatableMap: Map<string, string[]>    // Number conversations missing transcript for comparison <sourceName, conversationId[]>
```

Comment:
If the type of the keys is still string, what is the reason for using `Map` here?

I think the major benefit of using maps is when you need keys of non string type. If only string I wonder if you can use regular object for simpler operations.  I've been meaning to clean up the other instances where we use maps unnecessarily.

Can type as:
```
{ [key: string]: string[] }
OR
Record<string, string[]>
```

Reply:
Yup.  Good call.  This was in intermediate step.  They will be <string, string[]> of conversationIds